sdk/batch/Microsoft.Azure.Batch.FileStaging/src/SequentialFileStaging.cs

```diff
@@ -260,7 +286,7 @@ private static void CreateDefaultBlobContainerAndSASIfNeededReturn(List<IFileSta
                                                             policyName, 
                                                             startTime, 
                                                             expiredAtTime, 
-                                                            SharedAccessBlobPermissions.Read);
+                                                            "r");
```

Comment:
I take it there aren't enums for these in Track 2?

Reply:
Regarding the Shared access/blob permissions, I do not believe so. The converting construct in the Azure.Storage SDK is BlobSignedIdentifier which contains an AccessPolicy and accepts permissions as a string. However there BlobSasPermissions exist in Azure.Storage SDK which could be applicable: https://docs.microsoft.com/en-us/dotnet/api/azure.storage.sas.blobsaspermissions?view=azure-dotnet